libfreerdp/core/capabilities.c

```diff
@@ -3549,14 +3550,14 @@ static BOOL rdp_print_capability_sets(wStream* s, UINT16 numberCapabilities, BOO
 				break;
 		}
 
-		if (Stream_Pointer(s) != em)
+		rest = Stream_GetRemainingLength(&sub);
+		if (rest > 0)
 		{
 			WLog_ERR(TAG,
 			         "incorrect offset, type:0x%04" PRIX16 " actual:%" PRIuz " expected:%" PRIuz "",
-			         type, Stream_Pointer(s) - bm, em - bm);
+			         type, length + rest, length);
 		}
```

Comment:
suggestion: here instead of printing the offset which is useless we could just show the remaining bytes...

Reply:
what @hardening wrote, just show how many bytes were not handled

Reply:
already done in latest commit.